custom_model_runner/datarobot_drum/resource/components/Python/r_fit/fit.R

```diff
@@ -24,6 +24,69 @@ init <- function(code_dir) {
 }
 
 
+load_data <- function(input_filename){
+    tmp = readChar(input_filename, file.info(input_filename)$size)
+    data <- read.csv(text=gsub("\r","", tmp, fixed=TRUE))
+    data
```

Comment:
you could just return the read.csv line right

Reply:
yeah I think you're right - will confirm